### ISSUE TITLE
issue with JSZip, webpack and acorn

### ISSUE THREAD

**Issue by lisamcho** [Stuk/jszip#327]
There is an error when using acornJS with JSZip.

```
Error in plugin 'webpack-stream'
Message:
    ./~/jszip/package.json
Module parse failed: <PATH>/node_modules/jszip/package.json Unexpected token (2:9)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (2:9)
    at Parser.pp$4.raise (<PATH>/node_modules/acorn/dist/acorn.js:2221:15)
    at Parser.pp.unexpected (<PATH>/node_modules/acorn/dist/acorn.js:603:10)
    at Parser.pp.semicolon (<PATH>/node_modules/acorn/dist/acorn.js:581:61)
    at Parser.pp$1.parseExpressionStatement (<PATH>/node_modules/acorn/dist/acorn.js:966:10)
    at Parser.pp$1.parseStatement (<PATH>/node_modules/acorn/dist/acorn.js:730:24)
    at Parser.pp$1.parseBlock (<PATH>/node_modules/acorn/dist/acorn.js:981:25)
    at Parser.pp$1.parseStatement (<PATH>/node_modules/acorn/dist/acorn.js:709:33)
    at Parser.pp$1.parseTopLevel (<PATH>/node_modules/acorn/dist/acorn.js:638:25)
    at Parser.parse (<PATH>/node_modules/acorn/dist/acorn.js:516:17)
    at Object.parse (<PATH>/node_modules/acorn/dist/acorn.js:3098:39)
```

Commenting out this line from jszip/lib/index.js prevents the gulp error. 
`JSZip.version = require('../package.json').version;`

**Comment by elrumordelaluz** [comment]
Same here.
```
ERROR in ./~/jszip/package.json
Module parse failed: /Users/.../node_modules/jszip/package.json Unexpected token (2:8)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (2:8)
```
Solved with [json-loader](https://www.npmjs.com/package/json-loader)

in `webpack.config.js`
```js
//...
{ test: /\.json$/, loader: "json-loader" }
//...
```

**Comment by esfourteen** [comment]
broke for us as well with 3.1.0, 3.0.0 works fine

**Comment by dduponchel** [comment]
This comes from #317 where I reused the version in `package.json`.
I don't know well webpack: is there an easy fix on JSZip's side ? (if not, I'll revert to an hard-coded version).

**Comment by elrumordelaluz** [comment]
@dduponchel  Where is used precisely the _exposed_ `JSZip.version`?

**Comment by dduponchel** [comment]
It's not used internally, it is here to expose the version to other developers.

**Comment by lisamcho** [comment]
I also used json-loader within my webpack startup config but I still get that error.
`{test: /\.json$/, include: /node_modules/, loader: 'json-loader'},`

**Comment by lisamcho** [comment]
I also just checked that this does not occur when I pinned the version to 3.0.0.

**Comment by dduponchel** [comment]
Fixed in 3.1.1.
